salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc

```diff
@@ -0,0 +1,103 @@
+= SAP S/4HANA Order Management Guide
+:salesforce-oms-version: {page-component-version}
+
+== Overview
+
+This guide helps you create a sales order, freight order, outbound delivery, and invoice in SAP so that you can implement the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/Salesforce%20OMS%20to%20ERP%20Integration/[Salesforce OMS to ERP Integration] use case.
+
+== Before You Begin
+
+Submit a ticket on https://me.sap.com/[SAP for Me] to request activation of the following scope items in your Central Business Configuration (CBC) provisioned S/4HANA Cloud Essentials tenant:
+
+* Manual Transportation Planning (6W2)
+* Transportation Execution (6W3)
+* Freight Settlement (6W4)
+
+For more information about how to complete the ticket, see the https://userapps.support.sap.com/sap/support/knowledge/en/3122430[Scope item activation request for CBC provisioned S/4HANA Cloud Essentials tenant] knowledge base article on the SAP support site.
+
+=== Generated SAP Documents
+
+Completing the steps in this guide generates following SAP documents:
+
+* *Sales Order*
++
+A contractual agreement between a sales organization and a sold-to party to deliver products or provide a service at a set price, quantity, and time.
+* *Freight Order*
++
+The order that a carrier or shipper plans to execute, containing the logistical processing plan, vehicle departure times, and order execution data.
+* *Outbound Delivery Order*
++
+Contains the data that is required to trigger and monitor the outbound delivery process.
+* *Billing Document*
++
+The invoice that is created for an outbound delivery and that requests payment from the customer who placed the order.
```

Comment:
Suggestion: The invoice (that is) created for an outbound delivery and requests payment from the customer who placed the order.